tests/Backend/Workspaces/WorkspacesLoadTest.php

```diff
@@ -103,15 +102,23 @@ public function testWorkspaceLoadData()
         $input = [$mapping1, $mapping2];
 
         // test if job is created and listed
-        $initialJobs = $this->_client->listJobs();
         $runId = $this->_client->generateRunId();
         $this->_client->setRunId($runId);
+        $this->workspaceSapiClient->setRunId($runId);
+
         $workspaces->loadWorkspaceData($workspace['id'], ["input" => $input]);
-        $afterJobs = $this->_client->listJobs();
 
+        $afterJobs = array_filter(
```

Comment:
Tohle je tu už podruhé. Ale dejme tomu.

Reply:
Vyřeším v part2 https://github.com/keboola/storage-api-php-client/pull/568#issuecomment-772332837 tam se dost listujou joby workspaců tak to tam můžu upravit.